src/signals/incident-management/selectors.js

```diff
@@ -86,28 +100,30 @@ export const makeSelectEditFilter = createSelector(
   }
 );
 
-export const makeSelectFilterParams = createSelector(selectIncidentManagementDomain, incidentManagementState => {
-  const incidentManagement = incidentManagementState.toJS();
-  const filter = incidentManagement.activeFilter;
-  const { options } = filter;
-  const { page } = incidentManagement;
-  let { ordering } = incidentManagement;
-
-  if (ordering === 'days_open') {
-    ordering = '-created_at';
-  }
-
-  if (ordering === '-days_open') {
-    ordering = 'created_at';
-  }
+const filterParamsMap = {
+  area: 'area_code',
+  areaType: 'area_type',
+};
+const mapFilterParam = param => (filterParamsMap[param] ? filterParamsMap[param] : param);
 
+export const makeSelectFilterParams = createSelector(selectIncidentManagementDomain, incidentManagementState => {
+  const { activeFilter: filter, ordering, page } = incidentManagementState.toJS();
+  const orderingWithDaysOpen = ordering === 'days_open' ? '-created_at' : ordering;
```

Comment:
Setting two different variables doesn't make the code more readable. What was wrong with the conditions prior to this change?

Reply:
`let`, and I'm not allowed to use nested ternaries 😛

Reply:
I do agree that it's not too readable like this. I changed it to something that makes more sense.